lib/archive.go

```diff
@@ -122,15 +127,27 @@ func ReadArchive(in io.Reader) (*Archive, error) {
 
 		switch hdr.Name {
 		case "metadata.json":
-			if err := json.Unmarshal(data, &arc); err != nil {
+			if err = json.Unmarshal(data, &arc); err != nil {
 				return nil, err
 			}
 			// Path separator normalization for older archives (<=0.20.0)
-			arc.Filename = NormalizeAndAnonymizePath(arc.Filename)
-			arc.Pwd = NormalizeAndAnonymizePath(arc.Pwd)
+			if arc.K6Version == "" {
```

Comment:
hmm this check should probably be `arc.K6Version == "" || arc.K6Version == "0.24.0"`, given that the PR adding the version field has been merged for a few weeks, while this PR would probably be merged directly before 0.25.0

Reply:
this means that I need to up the version in this PR as well

Reply:
why?

Reply:
Because I use it to find out I don't need to normalize and anonymize the path and pwd again, which in general doesn't play well with URLs(especially the '://' part), and we run tests. Also in this particular case the check is okay because we have been normalizing and anonymizing paths and pwds for way longer than 0.24.0 and the addition of the version ;). Just previously it didn't matter

Reply:
Hmm, I still don't understand, sorry. Given that we'd merge this PR just before k6 0.25 is tagged and released, maybe by a matter of minutes, and we have had the archives with `0.24.0` for a version for ~20 days in master now, why shouldn't the check include `0.24.0`?

Reply:
again two things:
1. The code that this will run if there is version is not there for any archive that was created recently (with 0.24.0 for example) but for archives from long ago when normalizaiton and anomization hasn't been around. This whole code could probably be removed as I find it unlikely for such archives to be in the wild still. Although it doens't really hurt us as well
2. With the way filename and pwd are with this patch(URLS as strings) normalizing them again WILL break them.

Reply:
Ah, it makes sense now :tada: :smile: lists and punctuation FTW :stuck_out_tongue: